### DISCUSSION TITLE
Let's get StartStruck  🌟

### DISCUSSION THREAD

**Discussion by HasanSafarli** [Schweinepriester/github-profile-achievements#692] [General]
Star my project and comment below, I will star yours too:

https://github.com/HasanSafarli/food-delivery-app

**Comment by itspasindu** [comment]
done. 
https://github.com/itspasindu/isipathana-college-demo-site

Give me to star 😊

**Reply by HasanSafarli** [reply]
done :)

**Comment by ArielMAJ** [comment]
@HasanSafarli I starred you, please star my repo too: https://github.com/ArielMAJ/test.me
Feel free to contribute as well, there are many somewhat "simple" issues.

**Comment by Veercodeprog** [comment]
https://github.com/Veercodeprog/full-stack-assignment. done

**Comment by DevElkami** [comment]
Hi everyone,

[Password Vault](https://github.com/DevElkami/PasswordVault) is a personal password vault with firefox password importer. This vault use Advanced Encryption Standard (AES) encryption (MAUI and Winform).

I would appreciate it if you could check it out and give it a star. Thank you.
